packages/card/test.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable no-undef */
+import tap, { test, beforeEach, teardown } from 'tap';
+import { chromium } from 'playwright';
+
+tap.before(async () => {
+  const browser = await chromium.launch({ headless: true });
+  tap.context.browser = browser;
+});
+
+beforeEach(async (t) => {
+  const { browser } = t.context;
+  const context = await browser.newContext();
+  t.context.page = await context.newPage();
+});
+
+teardown(async () => {
+  const { browser } = tap.context;
+  browser.close();
+});
+
+test('Card component with no attributes is rendered on the page', async (t) => {
+  // GIVEN: A card component
+  const component = `
+    <f-card>
+      <div>This is an f-card</div>
+    </f-card>
+  `;
+
+  // WHEN: the component is added to the page
+  const { page } = t.context;
+  await page.setContent(component);
+  await page.addScriptTag({ path: './dist/index.js', type: 'module' });
+
+  // THEN:
+  const locator = await page.locator('f-card');
+  t.equal(
+    (await locator.innerHTML()).trim(),
+    '<div>This is an f-card</div>',
+    'HTML should be rendered',
+  );
+  t.equal(await locator.getAttribute('selected'), null, 'Selected attribute should be null');
+  t.equal(await locator.getAttribute('flat'), null, 'Flat attribute should be null');
+  t.equal(await locator.getAttribute('clickable'), null, 'Clickable attribute should be null');
+  t.equal(
+    await page.evaluate(
+      'document.querySelector("f-card").renderRoot.querySelector("div").tabIndex',
+    ),
+    -1,
+    'Tab index property should default to -1',
+  );
+  t.equal(
+    await page.evaluate(
+      'document.querySelector("f-card").renderRoot.querySelector("div").getAttribute("tabindex")',
+    ),
+    null,
+    'Tab index attribute should be undefined',
```

Comment:
Small thing - with `ifDefined(undefined)` I suppose the `tabindex` attribute will be removed?